app/assets/javascripts/discourse/app/static/prosemirror/extensions/link.js

```diff
@@ -130,6 +130,13 @@ const extension = {
   plugins: ({ pmState: { Plugin }, utils }) =>
     new Plugin({
       props: {
+        handleKeyDown(view, event) {
+          if (event.key === "k" && event.metaKey) {
+            // Avoids propagating this event to the chat global keydown handler
+            event.stopPropagation();
+            return false;
```

Comment:
is return false necessary here?

Reply:
The return of this method tells ProseMirror if we handled the event, so it can preventDefault.

I think returning true might break the actual mod-K keymap if the order of the plugins changes, but it's very unlikely.